test/check-machines-create

```diff
@@ -914,7 +914,8 @@ vnc_password= "{vnc_passwd}"
                 self.browser.click(".pf-c-modal-box__footer button:contains(Create and edit)")
             self.browser.wait_not_present("#create-vm-dialog")
 
-            self.goToVmPage(self.name)
+            if self.create_and_run:
+                self.goToVmPage(self.name)
```

Comment:
Also a separate bug fix. Please split out into separate PR, with a more detailed explanation, and test if possible. What does the "not redirected" mean?

Reply:
I think the first test which encountered this problem is for downloading RHEL images.
Also it's not a bug in the UI, just this function not being sufficient for test case `testCreateDownloadRhel`. So maybe it could be merged with other commit, like this one https://github.com/cockpit-project/cockpit-machines/pull/704/commits/765e5593fa1af9744d182ad74c6945432b7caa0f